pages/index.tsx

```diff
@@ -50,6 +50,7 @@ const IndexPage: React.SFC<IProps> = ({
     query: { candidate_id, role, user_id },
 }: IContext): Promise<IProps> => {
     let user;
+    user_id = user_id as string;
```

Comment:
Is it the good solution to cast user_id?

Reply:
no you should wrap it in if

```js

if (typeof user_id !== "string") {
throw new Error();
}
```